src/classes/RD2_DataMigration_TEST.cls

```diff
@@ -2370,4 +2446,26 @@ private class RD2_DataMigration_TEST {
         return Database.insert(new Opportunity(), false);
     }
 
+    /***
+    * @description Update two opportunities in the recurring donation to closed won for calculating rollups
+    */
+    private static void setFewOppsInRdToClosedWonForRollupTests(npe03__Recurring_Donation__c rd) {
+        List<Opportunity> oppToUpdate = new List<Opportunity>();
+        List<Opportunity> oppList =  oppGateway.getRecords(rd);
+
+        oppList.sort();
```

Comment:
What is this sorting by?

Reply:
It is sorting by dates, so that I update the first two opps to closed won.

Reply:
Is that defined somewhere as a custom sort interface? The default sorting of SObjects in a List collection is by the value in the "Name" field- https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_list_sorting_sobject.htm

Reply:
Its not custom sort here, when I debugged, it was sorting right, that might be because the opp name has the close date in it. 
If that might not be accurate, I can add a separate method in oppGateway to sort by close dates in this case.